src/IconToggle/index.js

```diff
@@ -21,31 +44,41 @@ export default class WrappedIconToggle extends Component {
       activeIcon,
       inactiveColor,
       activeColor,
+      toggleSize
     } = this.props
+    const defaultProps = {
```

Comment:
Oh, this is what you should add to the icon component to deal with backwards compatibility.